[HOLD until v1.0] Add script for MySQL in manual demo (#174)
openservicemesh/osm-docs: content/docs/getting_started/quickstart/manual_demo.md

kubectl apply -f https://raw.githubusercontent.com/openservicemesh/osm/release-v0.9/docs/example/manifests/apps/bookwarehouse.yaml
```

> The `bookwarehouse` project uses MySQL to store books data, thus we create `mysql` service account, service, StatefulSet and traffic policies.

Reviewer: ```suggestion
The `bookwarehouse` service uses MySQL to store books. Create resources for `mysql` service account, service, StatefulSet, and SMI traffic policies.
```
